main.rb

```diff
@@ -53,24 +52,23 @@ def main
 
 def welcome_screen
   welcome = "WELCOME TO THE TIC TAC TOE GAME!"
-  welcome = "*"*welcome.length + "\n" + welcome + "\n" + "*"*welcome.length
+  welcome = "#{'*' * welcome.length}\n#{welcome}\n#{'*' * welcome.length}"
 end
 
-def setup_player(num_of_player, name)
+def setup_player(num_of_player, default_name)
   print "Please insert the name of the player #{num_of_player}: "
   player_name = gets.strip.capitalize
-  player_name = player_name == "" ? name : player_name
+  player_name = player_name == "" ? default_name : player_name
```

Comment:
Is it `main`'s responsibility to sanitize this input? Is it really necessary? I think not.
Sanitizing is not bad per se but if you make `Player` protect against an empty name you can omit this.

If for some reason you still want to sanitize you could do something like:
`player_name = default_name if player_name.length < 1` which is easier to read/understand

Reply:
Resolved